recursion/intro-to-recursion.md

```diff
@@ -0,0 +1,114 @@
+# Intro to Recursion
+
+## Learning Goals
+
+- Define recursion
+
+## Introduction
+
+A powerful problem-solving technique in computer science is to use recursion. It takes advantage of operations that computers excel at.
```

Comment:
bold claims in this passage!
- a powerful technique (how?)
- is to use recursion (but what is it?) 
- operations that computers excel at (like???)

Instead, how about we keep it light with a bit of a ridiculous "definition," but one that can still get the reader thinking before we dive in further?

```suggestion
Life is full of problems. Big problems. But sometimes big problems are based on a smaller version of the same problem. And that smaller problem is itself based on a yet smaller version of the same problem. And that yet smaller problem is itself based on an even smaller version of the same problem, all the way down until we reach a size of the problem that isn't so big, and that we know how to solve.

Such problems are said to be recursive.
```